gtfs/spec/en/reference.md

```diff
@@ -550,6 +556,19 @@ Assigns stops from [stops.txt](#stopstxt) to areas.
 | `area_id` | Foreign ID referencing `areas.area_id` | **Required** | Identifies an area to which one or multiple `stop_id`s belong. The same `stop_id` may be defined in many `area_id`s. |
 | `stop_id` | Foreign ID referencing `stops.stop_id` | **Required** | Identifies a stop. If a station (i.e. a stop with `stops.location_type=1`) is defined in this field, it is assumed that all of its platforms (i.e. all stops with `stops.location_type=0` that have this station defined as `stops.parent_station`) are part of the same area. This behavior can be overridden by assigning platforms to other areas. |
 
+### area_sets.txt
+
+File: **Optional**
+
+Primary key (`*`)
+
+Groups collections of areas into sets.
+
+|  Field Name | Type | Presence | Description |
+|  ------ | ------ | ------ | ------ |
+| `area_set_id` | Unique ID | **Required** | Identifies an area set.  Must be unique in [area_sets.txt](#area_setstxt). |
```

Comment:
I assume an area_set can be a collection that contains multiple areas. A unique ID doesn’t seem suitable for mapping area to area_set?